extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css

```diff
@@ -81,18 +75,47 @@
   z-index: -1 !important;
   left: 50% !important;
   transform: translateX(-50%) rotate(45deg) !important;
+  border: 0.1em solid transparent !important;
   border-radius: 50% 50% 0% 50% !important;
   box-shadow: 0em 0em 0.5em rgba(0, 0, 0, 0.25) !important;
 }
 
 .i-amphtml-story-interactive-post-selection .i-amphtml-story-interactive-slider-bubble {
   transform: translate(var(--translate-x-center), -50%) scale(0.9) !important;
+  transition: transform var(--i-amphtml-interactive-animation-time)
+    var(--i-amphtml-interactive-ease-out-curve) !important;
 }
 
 .i-amphtml-story-interactive-post-selection .i-amphtml-story-interactive-slider-bubble::after {
   cursor: initial !important;
   border: 0.1em solid var(--i-amphtml-interactive-option-background-color) !important;
   border-bottom-right-radius: 50% !important;
+  transition: border-radius var(--i-amphtml-interactive-animation-time)
+      var(--i-amphtml-interactive-ease-out-curve),
+    border-color var(--i-amphtml-interactive-animation-time)
+      var(--i-amphtml-interactive-ease-out-curve) !important;
+}
+
+.i-amphtml-story-interactive-slider-bubble-wrapper {
+  /* Move the bubble up by sqrt(2)/2=70.7% to account for the pointy corner */
+  --scale-var: calc(var(--fraction) + 1) !important;
+  transform: translate(
+      calc(var(--thumb-size) * var(--fraction)),
```

Comment:
The arrow alignment is slightly off. It might be from the math here but I'm not sure.
<img width="190" alt="Screen Shot 2021-07-30 at 12 57 41 PM" src="https://user-images.githubusercontent.com/3860311/127686798-6d4d90ff-3747-40ad-820d-feca53b5fe7b.png">

Reply:
Subtracting `.1em` from `--thumb-size` in the x transform seems to help this:
`calc((var(--thumb-size) - .1em) * var(--fraction))`
I'm not sure why though. I removed the borders as Matias mentioned and also didn't see an effect.